test/browser_tests/spec_suites/newsroom.js

```diff
@@ -0,0 +1,137 @@
+'use strict';
+
+var Newsroom = require(
+    '../page_objects/page_newsroom.js'
+  );
+
+describe( 'The Newsroom Page', function() {
+  var page;
+
+  beforeAll( function() {
+    page = new Newsroom();
+    page.get();
+  } );
+
+  it( 'should properly load in a browser', function() {
+    expect( page.pageTitle() ).toBe( 'Newsroom' );
+  } );
+
+  it( 'should include a main title', function() {
+    expect( page.mainTitle.getText() ).toBe( 'Newsroom' );
+  } );
+
+  it( 'should have a side nav', function() {
+      expect( page.sideNav.isPresent() ).toBe( true );
+    }
+  );
```

Comment:
fixed in https://github.com/cfpb/cfgov-refresh/commit/071c9fc3f5c87f92f74c956a1b34fa0c0d665156